### ISSUE TITLE
False negative `no-value-for-parameter` when positional-only argument name consumed by **kwargs

### ISSUE THREAD

**Issue by jacobtylerwalls** [pylint-dev/pylint#8559]
### Bug description

```python
# pylint: disable=missing-function-docstring, missing-module-docstring
def required_positional_with_var_kwargs(a, /, **_kwargs):
    return a

required_positional_with_var_kwargs(a=43)
```

### Command used

```shell
pylint a.py
```

### Pylint output

```shell
nothing
```

### Expected behavior

no-value-for-parameter

### Pylint version

```shell
pylint 3.0.0b1
astroid 2.16.0dev0
Python 3.8.10
```
